src/beta_inc.jl

```diff
@@ -903,28 +919,28 @@ function beta_inc_inv(a::Float64, b::Float64, p::Float64, q::Float64; lb = logbe
 
     #Initial approx
 
-    r = sqrt(-log(pp^2))
+    r = sqrt(-2*log(pp))
     pp_approx = r - @horner(r, 2.30753e+00, 0.27061e+00) / @horner(r, 1.0, .99229e+00, .04481e+00)
 
     if a > 1.0 && b > 1.0
         r = (pp_approx^2 - 3.0)/6.0
         s = 1.0/(2*aa - 1.0)
         t = 1.0/(2*bb - 1.0)
-        h = 2.0/(s+t)
-        w = pp_approx*sqrt(h+r)/h - (t-s)*(r + 5.0/6.0 - 2.0/(3.0*h))
-        x = aa/ (aa+bb*exp(w^2))
+        h = 2.0/(s + t)
+        w = pp_approx*sqrt(h + r)/h - (t - s)*(r + 5.0/6.0 - 2.0/(3.0*h))
+        x = aa/(aa + bb*exp(w + w))
```

Comment:
So the `w^2` in the original code was a typo?

Reply:
It is a typo. This is the line from the paper
<img width="419" alt="Skærmbillede 2021-05-19 kl  20 34 20" src="https://user-images.githubusercontent.com/505001/118865866-b6441680-b8e1-11eb-9b3b-930d977d991c.png">